ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -58,12 +60,12 @@ function (
         for (i = 0; i < blocks.length; i++) {
             block = blocks[i];
 
-            if (i === 1 || i === 6) {
+            if (i === 2 || i === 7) {
                 numberOfMpus++;
                 mpu = createMpu(numberOfMpus);
 
                 if (block.nextSibling) {
-                    block.parentNode.insertBefore(mpu, block.nextSibling);
+                    block.parentNode.insertBefore(mpu, block);
```

Comment:
They do still go after the 2nd and 7th block. See this test: https://github.com/guardian/mobile-apps-article-templates/pull/621/files/a45f307dd8db9c79178f1207d3a51f03778fbc12#diff-dcf2f2e661a79bd5a44f16f89b56608aR269

Unless I have misunderstood the test